TicketSwap/solar
# Change Clock icon in line with Foundation design file

# Description
The Clock icon was not in line with the Figma design doc, so this PR replaces the path’s D property for the new icon.

## Changes


## How to test
- Checkout this branch
- `$ yarn dev`
- yarn storybook
- Open http://localhost:9001/?path=/story/guidelines-iconography--page and note that the icon is displayed correctly

## Screenshots
Old clock icon:
![image](https://user-images.githubusercontent.com/49148610/177592389-374992a9-543b-4f8e-a4fc-60a2b6ecf7b2.png)

New icon ([from this Figma file](https://www.figma.com/file/JZZe78lkpqqNKdrqHltAPh/%E2%A0%BD-%E2%9B%BA%EF%B8%8F---Foundation?node-id=1628%3A220))
![image](https://user-images.githubusercontent.com/49148610/177592436-da1e05da-a034-48f3-9f72-e3d8f36225c4.png)


## To be notified
@TicketSwap/frontend 

## Links
https://ticketswap.slack.com/archives/C47FK1ZUH/p1657119756808339
